### PR TITLE
apply precommit hooks

### PR DESCRIPTION
**Pull request by mahmoudmohamed22**
Apply pre-commit hooks in files project
    1- check-yaml
    2-End-of-file-fixer
    3-Trim Trailing-whitespace
    4-Debug-statements
    5- Double-quote-string-fixer
    6-isort
    7-NodeJS HTML syntax linter (htmlhint)

### CODE REVIEW

**Comment by mahmoudmohamed22** [comment]
@aselarbd @orionpax00  please review my pull request to developing a pre-commit hook for the repository

**Comment by mahmoudmohamed22** [comment]
@ivantha please review my pull request to developing a pre-commit hook for the repository

File: legacy/tensormap-server/app/resources/model_templates/keras_temp_multi_classification.py [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
+_x = dataCsv[0]
+_y = dataCsv[1]
+
+x_train, y_train, x_test, y_test = train_test_split(_x, _y, random_state=42, shuffle=True, test_size=0.10)
 
-network = tf.keras.models.Sequential(name='userModel')
+network = tf.keras.models.Sequential(name="userModel")
 
-network.compile (optimizer = ,loss = )
+network.compile(optimizer="adam", loss="Cross-entropy loss")
```
Comment (aselarbd) [inline legacy/tensormap-server/app/resources/model_templates/keras_temp_multi_classification.py:20]: Don't hard code optimizers and loss function in here. Create a variable on top of the file and assign the variable

File: legacy/tensormap-server/app/common/make_model_json.py [hunk 2]
```diff
@@ -1,41 +1,48 @@
-from .. import db
 import json
-from ..resources.database_models.code_gen import template_copies,user_template_index,code_layers
 
-def getNodeVal(layerId,sentKey,json_config):
+from .. import db
+from ..resources.database_models.code_gen import (
```
Comment (aselarbd) [inline legacy/tensormap-server/app/common/make_model_json.py:4]: bet not use relative paths

File: legacy/tensormap-server/app/common/make_model_json.py [hunk 3]
```diff
@@ -44,36 +51,35 @@ def makeKerasModel(json_config):
...
-    modelCommon["config"]["layers"][0]["input_shape"] =  [1000,]
+                break
+
+    layerCommon = {"name": "userModel", "layers": dict_list}
+    modelCommon = {
+        "class_name": "Sequential",
+        "config": layerCommon,
+        "keras_version": "2.2.4-tf",
+        "backend": "tensorflow",
+    }
```
Comment (aselarbd) [inline legacy/tensormap-server/app/common/make_model_json.py:75]: Make a constant parameter file and define these parameters there and invoke in here

File: legacy/tensormap-server/app/resources/model_templates/keras_temp_multi_classification.py (hunk 1 above)
Comment (mahmoudmohamed22) [reply legacy/tensormap-server/app/resources/model_templates/keras_temp_multi_classification.py]: it's done

Comment (mahmoudmohamed22) [reply legacy/tensormap-server/app/resources/model_templates/keras_temp_multi_classification.py]: it's done

File: legacy/tensormap-server/app/common/make_model_json.py (hunk 3 above)
Comment (mahmoudmohamed22) [reply legacy/tensormap-server/app/common/make_model_json.py]: it's done

Comment (mahmoudmohamed22) [reply legacy/tensormap-server/app/common/make_model_json.py]: it' Done

File: legacy/tensormap-server/app/resources/model_templates/keras_temp_multi_classification.py (hunk 1 above)
Comment (mahmoudmohamed22) [reply legacy/tensormap-server/app/resources/model_templates/keras_temp_multi_classification.py]: it's done

File: legacy/tensormap-server/app/common/make_model_json.py (hunk 2 above)
Comment (mahmoudmohamed22) [reply legacy/tensormap-server/app/common/make_model_json.py]: Do I convert it to absolute paths?

**Comment by mahmoudmohamed22** [comment]
I made the edits and push on branch is precommit-hooks
please review edits @aselarbd

**Comment by mahmoudmohamed22** [comment]
@ivantha @aselarbd

**Review by aselarbd — DISMISSED** [review]
Looks good

**Comment by aselarbd** [comment]
can you rebase with master ?

**Comment by mahmoudmohamed22** [comment]
@aselarbd  review branch rebased master

**Review by ivantha — APPROVED** [review]

**Review by ivantha — APPROVED** [review]
